store/localstore/boltdb/boltdb.go

```diff
@@ -40,20 +39,37 @@ func (d *db) Get(key []byte) ([]byte, error) {
 
 	err := d.DB.View(func(tx *bolt.Tx) error {
 		b := tx.Bucket(bucketName)
-		v := b.Get(key)
-		if v == nil {
-			return nil
+		value = b.Get(key)
```

Comment:
The value is only valid in View transaction, we should clone it for outer use, see boltdb manual https://github.com/boltdb/bolt#using-keyvalue-pairs